source/util.js

```diff
@@ -63,3 +65,16 @@ exports.getTagVersionPrefix = pMemoize(async options => {
 		return 'v';
 	}
 });
+
+function splitFileNameList(stdout) {
```

Comment:
No need to create a method for this IMO, it can be a part of `getNewFilesIgnoredByNpm`.

Reply:
is moved in `git-util`. The function wasn't almost correct. I fixed it by the way.